go.sum

```diff
@@ -451,6 +449,8 @@ github.com/imdario/mergo v0.3.10/go.mod h1:jmQim1M+e3UYxmgPu/WyfjB3N3VflVyUjjjwH
 github.com/imdario/mergo v0.3.11/go.mod h1:jmQim1M+e3UYxmgPu/WyfjB3N3VflVyUjjjwH0dnCYA=
 github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANytuPF1OarO4DADm73n8=
 github.com/j-keck/arping v0.0.0-20160618110441-2cf9dc699c56/go.mod h1:ymszkNOg6tORTn+6F6j+Jc8TOr5osrynvN6ivFWZ2GA=
+github.com/jjbustamante/imgutil v0.0.0-20210817191209-1eb46031019f h1:mP9fcRmifFuCZh5irWfdr2MKJKTXdCQs2hdh04hd80U=
+github.com/jjbustamante/imgutil v0.0.0-20210817191209-1eb46031019f/go.mod h1:u67HqXr7QbMDRXn8plSamdI9LgYu1kODpIp0c6LoCDE=
```

Comment:
Do we still need it?

Reply:
Not sure if the [PR #126](https://github.com/buildpacks/imgutil/pull/126) on imgUtil repo was already merged, if it was, I will delete those lines

Reply:
You're right. I thought it was merged yesterday.

Reply:
Removed at [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/7ce8bb8297e15af0aab609e3df4c462172972ef9)